packages/design-picker/src/components/theme-preview-container/style.scss

```diff
@@ -0,0 +1,104 @@
+@import '@automattic/onboarding/styles/mixins';
+
+.theme-preview-container {
+	display: flex;
+	flex-direction: row;
+}
+
+.theme-preview-container__sidebar {
+	width: 343px;
+
+	> div {
+		margin: 2rem 0 0;
+
+		h2 {
+			color: var( --studio-gray-100 );
+			font-size: $font-body-small;
+			font-weight: 500;
+			margin-bottom: 0.5rem;
+		}
+	}
+
+	.theme-preview-container__sidebar-title {
+		margin: 0 0 16px;
```

Comment:
I think we could move the margin to the `margin-top` of the `theme-preview-container__sidebar-description` element. Otherwise, the spacing looks weird as it becomes 16px + 32px